src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
Instead of using `toString()` to get the string representation of the enum constants, use the `name()` method. According to the checklist, `name()` is preferred because it always returns the string representation of the specific enum constant, whereas `toString()` can be overridden and may not return the expected value.